parallel-install/pkg/logger/logger.go

```diff
@@ -0,0 +1,37 @@
+package logger
+
+import (
+	"go.uber.org/zap"
```

Comment:
Again, this makes our library hard-coded to a single logger. What if the client has other preferences? Well, now there are two loggers in the project. What if CLI decides they need some other logger? We'd have to make another PR touching 20 files or so. Take a look at how Helm is solving that - they're using an abstract log function. The current logging is based on the same idea.
Personally, I see this as a step backward in the usability of the library.

Reply:
After followup, we have decided to change the strategy. Now logger will be created once on the highest level and passed around.